#the process noise for the robot motion model
std_q: [0.5,0.5,0.5]
#the noise in measurement model
std_r: [1]
#max measurements 
max_num: 10
#dist_thresh_for_cluster 0.4 0.15 90 140
#real robot  0.1 0.50 40 160
dist_thresh: 0.4
std_angle: 0.15
min_mean_angle: 90
max_mean_angle: 140
radius_range: 0.13
#whether to add new landmark
new_landmark_thresh: 0.02